sendWhatsappDm.py

```diff
@@ -56,7 +56,7 @@
     print('Sending message to:', phone)
     try:
         #opening whatsapp chat for the current number
-        CHAT_URL = "https://web.whatsapp.com/send?phone={phone}&text&type=phone_number&app_absent=1"
+        CHAT_URL = f"https://web.whatsapp.com/send?phone={phone}&text&type=phone_number&app_absent=1"
         browser.get(CHAT_URL.format(phone=phone))
         time.sleep(1)
 
```

Comment:
<img width="839" alt="Screenshot 2024-04-14 at 12 49 35 AM" src="https://github.com/theshdb/whatsapp-dm/assets/64187899/3b6c0e3e-a1fc-4a7b-a300-1355188cf56a">


The above value can also be fetched from `constants.py`